### PR TITLE
feat: Add edit api token

### PR DESCRIPTION
**Pull request by killbond007**
## Description

Add edit api token

Capture d’écran 2021-04-14 à 20 56 59

### CODE REVIEW

File: src/shared/api-tokens/api-tokens-resolver.ts [hunk 1]
```diff
@@ -32,23 +26,26 @@ class UpsertTokensRequest {
   _id: string;
 
   @Field(() => String, { nullable: true })
-  sendgridApiKey?: string;
-
-  @Field(() => String, { nullable: true })
-  carrierToken?: string;
+  launchRoomToken?: string;
```
Comment (bitQ2019) [inline src/shared/api-tokens/api-tokens-resolver.ts:29]: why token can be null?

**Review by bitQ2019 — CHANGES_REQUESTED** [review]
The UI confuses me.
Maybe we can use a label to display the token and an editing button to update token

File: src/shared/api-tokens/api-tokens-resolver.ts (hunk 1 above)
Comment (killbond007) [reply src/shared/api-tokens/api-tokens-resolver.ts]: ok , i remove nullable

**Review by bitQ2019 — APPROVED** [review]

**Comment by bitQ2019** [comment]
> The UI confuses me.
> Maybe we can use a label to display the token and an editing button to update token

how about this?

**Review by puncsky — APPROVED** [review]

**Review by puncsky — APPROVED** [review]

**Pull request merged**